docs/topics/vscode-extension-configuring-analysis.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-configuring-analysis_{context}']
+
+= Configuring a run configuration
+
+You can configure a number of separate run configurations to run against each of your projects. Each configuration is saved automatically as you work.
+
+.Prerequisites
+
+* Install `mta-cli`.
+* Projects imported into {VscodePluginShortName}.
+
+.Procedure
+
+. In *Extensions* view, click the *{ProductName}* icon (image:vs_MTA_extension_icon.png[MTA code extension]) on the Activity bar.
+. If you do not have the MTA CLI installed, click *Download* on the *Unable to find MTA CLI* window.
+. Click the *+* (plus sign) next to *{ProductName}* to add a run configuration.
+. Optional: Enter a new *Name*.
+. At a minimum, complete the following configuration fields:
```

Comment:
Remove "At a minimum". Either the user must complete those fields or not. If they are mandatory, then just say "Complete the following fields"